main.go

```diff
@@ -91,6 +92,7 @@ func init() {
 	flag.StringVar(&filePrefix, "prefix", "", "prefix: file prefix for generated files")
 	flag.StringVar(&driverName, "driver", "pg_dump", "driver name: flag for accessing source DB or dump files (accepted values are \"pg_dump\", \"postgres\", \"mysqldump\", and \"mysql\")")
 	flag.Int64Var(&schemaSampleSize, "schema-sample-size", int64(100000), "schema-sample-size: the number of rows to use for inferring schema (only for DynamoDB)")
+	flag.BoolVar(&consistentRead, "consistent-read", false, "consistent-read: enable the 'consistent read' mode when scanning data but it causes more api quota usage (only for DynamoDB)")
```

Comment:
I'm not sure I understand the value of using strongly consistent reads here.

If I'm running HarbourBridge against a live dynamodb instance, then I'm going to effectively get a mix of data from different time instances: schema processing, and each table query for data conversion will be issued at different times. And if I'm reading a large table, then I might see inconsistencies across the contents of the same table in the sense that rows I get later could be from more recent writes than rows I read earlier. Using strongly consistent reads won't really help much.

Note that the time to reach consistency is typically about one second or less -- see https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ReadConsistency.html.

If my dynamodb instance isn't changing, then it probably doesn't matter what I do.

However, the cost of strongly consistent reads can be quite high.

I guess my concern with the flag is that it could give a false sense of security: I set the flag thinking it will help me get some more consistent view of the data, but it won't really help and it will cost me.

Let's chat more offline on this.